roxie/ccd/ccdserver.cpp

```diff
@@ -27366,6 +27409,69 @@ class CPseudoActivity : public CRoxieServerActivity
     }
 };
 
+class CSequentialSinkSet : public CRoxieServerActivity
```

Comment:
CRoxieServerPrefetchProjectActivity should return false (always? if parallel is set?)

Reply:
Other activities:
- All versions of Soapcall should return false.
- Whatever is used for roxiepipe. (TAKpipe?
- CRoxieServerExternalActivity?
- Will versions of index operations (count/aggregate) that go remote be included?  Should they be?

I wonder if isSimpleSink() should be renamed to be more descriptive.  E.g.

isWorthExecutingInParallel()   canBlockThread()  containsRemoteCall() or something similar.